tools/generator/src/Generator/CreateFilesAndGroups.swift

```diff
@@ -568,11 +568,15 @@ extension Generator {
                     }
                     .uniqued()
 
+                var onceFilePaths: Set<FilePath> = []
+                var onceOtherFlags: Set<String> = []
                 let clangOtherArgs = try lldbContext.clang.map { clang in
```

Comment:
This map now performs mutations as a side-effect, so it might be clearer to change this to a loop since this is no longer purely functional. I don't feel strongly though.